src/zenml/cli/served_models.py

```diff
@@ -418,4 +419,7 @@ def get_model_service_logs(
         if raw or line.startswith("\x1b["):
             print(line)
         else:
-            console.print(line)
+            try:
+                console.print(line)
+            except MarkupError:
+                print(line)
```

Comment:
Which `print` does this use? `rich` print or the Python print?

Reply:
@strickvl  That's python print as rich fail to format it! i couldn't find a way of how to do it with rich print

Reply:
Got it. If it's not too much extra work, it'd be nice to reproduce this error in a test, but otherwise the fix looks ok to me :)